pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/PulsarAdminTool.java

```diff
@@ -205,23 +205,17 @@ public static void main(String[] args) throws Exception {
         }
 
         ++cmdPos;
-        boolean isLocalRun = cmdPos < args.length && "localrun".equals(args[cmdPos].toLowerCase());
 
         Function<PulsarAdminBuilder, ? extends PulsarAdmin> adminFactory;
```

Comment:
Function<PulsarAdminBuilder, ? extends PulsarAdmin> adminFactory  = (adminBuilder) -> {  };
Is this also possible?

Reply:
It seems impossible. When I test, report missing return value in lambda expression, Causing compilation errors